website/_docs/howto/howto_upgrade.md

```diff
@@ -1,112 +1,146 @@
 ---
 layout: docs
-title:  Upgrade From Old Versions
+title:  Upgrade Kylin From Old Versions
 categories: howto
 permalink: /docs/howto/howto_upgrade.html
 since: v1.5.1
 ---
 
-Running as a Hadoop client, Apache Kylin's metadata and Cube data are persistended in Hadoop (HBase and HDFS), so the upgrade is relatively easy and user does not need worry about data loss. The upgrade can be performed in the following steps:
+Running as a Hadoop client, Apache Kylin's metadata and Cube data are persistent in Hadoop (HBase and HDFS), so the upgrade is relatively easy and user does not need worry about data loss. The upgrade can be performed in the following steps:
```

Comment:
"persistent" is incorrect in this context, it is adjective, while here we need "be persisted" which is a passive voice;  (a typo in the previous version).